src/send/mod.rs

```diff
@@ -2,6 +2,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
 
+#![cfg_attr(feature = "cargo-clippy", allow(needless_pass_by_value))]
```

Comment:
I'm wary of module-level allows in case they permit future us to do something silly. Which code necessitates this one and is there a way to target it more locally to that specific line or whatever?

Reply:
We can limit this only to the function that we want to bypass. I'll change that.